clinica/iotools/converters/oasis_to_bids/oasis_to_bids.py

```diff
@@ -95,14 +91,41 @@ def convert_single_subject(subj_folder):
                 os.mkdir(path.join(session_folder))
                 os.mkdir(path.join(session_folder, 'anat'))
 
-            # In order do convert the Analyze format to NIFTI the path to the .img file is required
+            # In order do convert the Analyze format to Nifti the path to the .img file is required
             img_file_path = glob(path.join(t1_folder, '*.img'))[0]
             output_path = path.join(session_folder, 'anat',
                                     bids_id + '_ses-M00_T1w.nii.gz')
-            subprocess.run('mri_convert' + ' ' + img_file_path + ' ' + output_path,
-                           shell=True,
-                           stdout=subprocess.DEVNULL,
-                           stderr=subprocess.DEVNULL)
+
+            # First, convert to Nifti so that we can extract the s_form with NiBabel
+            # (NiBabel creates an 'Spm2AnalyzeImage' object that does not contain 'get_sform' method
+            img_with_wrong_orientation_analyze = nb.load(img_file_path)
+
+            # OASIS-1 images have the same header but sform is incorrect
+            # To solve this issue, we use header from images converted with FreeSurfer
+            # to generate a 'clean hard-coded' header
+            # affine:
+            # [[   0.    0.   -1.   80.]
+            #  [   1.    0.    0. -128.]
+            #  [   0.    1.    0. -128.]
+            #  [   0.    0.    0.    1.]]
+            affine = np.array([0, 0, -1, 80, 1, 0, 0, -128, 0, 1, 0, -128, 0, 0, 0, 1]).reshape(4, 4)
+            s_form = affine.astype(np.int16)
+
+            hdr = nb.Nifti1Header()
+            hdr.set_data_shape((256, 256, 160))
+            hdr.set_data_dtype(np.int16)
+            hdr['bitpix'] = 16
+            hdr.set_sform(s_form, code='scanner')
+            hdr.set_qform(s_form, code='scanner')
+            hdr['extents'] = 16384
+            hdr['xyzt_units'] = 10
+
+            img_with_good_orientation_nifti = nb.Nifti1Image(
+                np.round(img_with_wrong_orientation_analyze.get_data()).astype(np.int16),
```

Comment:
> Should we inform users of this change of data type? Or it's maybe irrelevant?

I used this data type conversion because `mri_convert` from FreeSurfer produced Nifti files with int16 type. In practice, I think the `astype(np.int16)` could be removed but I wanted to be close as possible of the conversion with FreeSurfer.